runtime/runtime_controller.h

```diff
@@ -660,6 +660,24 @@ class RuntimeController : public PlatformConfigurationClient {
       std::shared_ptr<PlatformIsolateManager>(new PlatformIsolateManager());
   bool has_flushed_runtime_state_ = false;
 
+  // Tracks the views that have been called `Render` during a frame.
+  //
+  // If all registered views (through `AddView`) have been rendered, then the
+  // end of frame will be called immediately, submitting the views to the
+  // pipeline a bit earlier than having to wait for the end of the vsync.
```

Comment:
This waited until the end of the `onDrawFrame` callback, right? The end of the vsync is a separate event that happens later.

Consider something like:

```suggestion
  // pipeline a bit earlier than having to wait for the end of the `onDrawFrame` callback.
```

Reply:
I think they're more or less the same event. But anyway, the vsync is not a concept of the `RuntimeController`. I'm changing the doc to this, what do you think?
```cpp
  // Tracks the views that have been called `Render` during a frame.
  //
  // If all views that have been registered by `AddView` have been called
  // `Render`, then the runtime controller notifies the client of the end of
  // frame immediately, allowing the client to submit the views to the pipeline
  // a bit earlier than having to wait for the end of `BeginFrame`. See also
  // `Animator::OnAllViewsRendered`.
  //
```